packages/frontend-2/lib/viewer/composables/setup/postSetup.ts

```diff
@@ -139,7 +139,7 @@ function useViewerObjectAutoLoading() {
 
       await Promise.all(removableObjectIds.map((i) => loadObject(i, true)))
       await Promise.all(
-        addableObjectIds.map((i) => loadObject(i, false, { zoomToObject }))
+        addableObjectIds.map((i) => loadObject(i, false, { zoomToObject: false }))
```

Comment:
This PR seems incomplete. Where's the vue template changes? Where's the cookie? Where's the part where you read the cookie and use it in this fn call?

Reply:
Sorry Fabs, the brief for this one changed, I should have updated the ticket with the new brief. I've updated the PR description now. 

It was decided to drop the Toggle, and instead add a Call to action to the new Toast that appears on a new model being added. 

We also don't zoom when a new model is added to viewer now, only when the user clicks the call to action.

Reply:
Uploading Screen Recording 2024-06-07 at 11.38.09.mov…

Here's a quick video to save you loading this up